ex2_hal/adcs/equipment_handler/src/adcs_handler.c

```diff
@@ -2130,7 +2309,7 @@ ADCS_returnState ADCS_get_log_config(uint8_t *flags_arr, uint16_t *period, uint8
 
     for (int j = 0; j < 10; j++) {
         for (int i = 0; i < 8; i++) {
-            *(flags_arr + (8 * j) + i) = (telemetry[j] >> i) & 1;
+            *(*flags_arr + (8 * j) + i) = (telemetry[j] >> i) & 1;
```

Comment:
Why did you make flags_arr uint8_t**? It looks like it was fine as uint8_t*

Reply:
We have confirmed flags_arr appears to work in binary testing as a double pointer, although the function right now results in wrong endianness.

Reply:
The change you made has no semantic difference to the original. I think that means you were probably calling it incorrectly, e.g. with &foo instead of foo. Pointers are hard :-/

Reply:
I realize this is under the ADCS_get_log_config() function, my previous comment was on the set_log_config() command. The ADCS_get_log_config() function was confirmed to work accurately in binary testing as it is right now.

I should clarify, I did not actually make this change, so I cannot comment on why this change was made.